test/test_parse_logcfg.c

```diff
@@ -915,6 +916,12 @@ void test_ssbmode(void **state) {
     assert_int_equal(trxmode, SSBMODE);
 }
 
+void test_operation_mode(void **state) {
```

Comment:
double check the spelling of the function, the parameter is called `OPERATING_MODE`. Check also the man page.